modules/library/jdbc/src/test/java/org/geotools/jdbc/JDBCGeometryOnlineTest.java

```diff
@@ -41,40 +45,49 @@ public abstract class JDBCGeometryOnlineTest extends JDBCTestSupport {
     @Override
     protected abstract JDBCGeometryTestSetup createTestSetup();
 
+    @Test
```

Comment:
The ``@SuppressWarnings("PMD.JUnit4TestShouldUseTestAnnotation")`` annotation can be removed in this class too.

Reply:
Done